exim/manifest.json

```diff
@@ -0,0 +1,35 @@
+{
+  "display_name": "exim",
+  "maintainer": "JeanFred1@gmail.com",
+  "manifest_version": "1.0.0",
+  "name": "exim",
+  "metric_prefix": "exim.",
+  "metric_to_check": "exim.queue.count",
+  "creates_events": false,
+  "short_description": "exim integration to monitor mail queues",
+  "guid": "04552816-c3e6-4a0a-90a0-bd751633984d",
+  "support": "contrib",
+  "supported_os": [
+    "linux",
+    "mac_os",
+    "windows"
+  ],
+  "public_title": "exim",
+  "categories": [
+    ""
```

Comment:
These are the categories on this page, see if any one fits https://docs.datadoghq.com/integrations/

Reply:
Picked `Collaboration`, like sendmail and postfix